tests/build.gradle

```diff
@@ -64,21 +77,14 @@ ext.testSets = [
         ]
     ],
     "REQUIRE_SYSTEM" : [
-        "includes" : [
-            "apigw/healthtests/**",
-            "services/**",
-            "system/basic/**",
-            "system/rest/**",
-            "whisk/core/admin/**",
-            "whisk/core/cli/test/**",
-            "whisk/core/apigw/actions/test/**",
-            "whisk/core/limits/**",
-            "whisk/core/database/test/*CacheConcurrencyTests*",
-            "whisk/core/controller/test/*ControllerApiTests*",
-        ]
+        "includes" : systemIncludes
     ],
     "LEAN" : [
         "excludes" : leanExcludes
+    ],
+    "REQUIRE_LEAN_SYSTEM" : [
+        "includes" : systemIncludes,
+        "excludes" : ["**/*KafkaConnectorTests*"]
```

Comment:
A comment would be nice, stating why you exclude these.

Reply:
Comment added